frontend/src/components/atoms/IconText.tsx

```diff
@@ -12,7 +12,7 @@ const IconText = ({ icon, text }: IconTextProps) => {
   return (
     <>
       <div className="flex items-center">
-        <div className="flex-left">{icon}</div>
+        <div className={"flex-left"}>{icon}</div>
```

Comment:
I don't believe there is a need to wrap `flex-left` in curly braces?

Reply:
Oh yes! I thought that I reverted the changes but it seems that the code wasn't pushed. I will fix this along with the modifications made to IconText. :)